src/mapping.conf

```diff
@@ -67,9 +67,11 @@
         "battery.voltage"       :   "voltage.battery",
 
         "outlet.realpower"      :   "realpower.default",
+        "outlet.power"          :   "power.default",
```

Comment:
Also beware that there is a method NUTDevice::NUTRealpowerFromOutput in nut_device.cc that does the fallback according to what is available